### DISCUSSION TITLE
[Q&A] 问一下这个toolstoken有什么作用呢？

### DISCUSSION THREAD

**Discussion by scortLi** [lobehub/lobehub#1080] [General Question] [answered]
问下这个toolstoken有什么作用呢，这个值是固定的吗还是在哪里可配置，为什么要限定这个值呢
![image](https://github.com/lobehub/lobe-chat/assets/14088043/71df3c06-f4dc-4393-9a42-98ccd1fb96d9)

**Comment by arvinxx** [comment, marked as answer]
这个代表了当前会话窗口下还能使用的 token 数，超过这个 token 就没法进行会话了。

**Reply by scortLi** [reply]
@arvinxx 那这个值可以修改吗或者在哪里可以配置吗

**Reply by arvinxx** [reply]
不可以改，这个值和模型本身能力有关系。比如 gpt-3.5-turbo-0314 的最大会话窗口是 4k， gpt-4-1106-preview 的会话窗口是 128k。 4k 窗口的模型，在界面上改到 16k 也没用，传过去发到模型还是会报错。

**Reply by scortLi** [reply]
那是不是我把会话消息设置成1.每次会话结束之后这个token值就会变回原来的值

**Reply by arvinxx** [reply]
是的
